src/ex3/PokemonClient.js

```diff
@@ -0,0 +1,13 @@
+import fetch from "node-fetch"
+
+export default class PokemonClient {
+  constructor() {
+    this.api = "https://pokeapi.co/api/v2/pokemon/"
```

Comment:
small improvement - this string is better defined as const because it doesn't change, rather than an instance var